cmd/deploy/deploy.go

```diff
@@ -541,6 +541,7 @@ func shouldRunInRemote(opts *Options) bool {
 		return true
 	}
 
+	oktetoLog.Information("Have you tried the `--remote` option? You can run the deploy commands in your Okteto cluster, check our docs for more.")
```

Comment:
Could we add a link to the `remote` docs?

Reply:
👍 I'll do! First of all, I would like to clarify if it is necessary to add more documentation about the feature to know which link is the right one to use in this suggestion. I think for now we have what we need but I would like to know your opinion. For now we can link or use from the [cli](https://www.okteto.com/docs/reference/cli/#deploy:~:text=bash%20(defaults%20to%20false)-,%2D%2Dremote,-(bool)) or in the [manifest](https://www.okteto.com/docs/reference/manifest/#deploy-remotely)

Reply:
I pointed to the docuemntation of the manifest which shows the detail about the feature. 91423593ff7f8379dc7dcd9702b30f5d5619b51f